SimTracker/TrackTriggerAssociation/plugins/CleanAssoc.cc

```diff
@@ -0,0 +1,83 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/Run.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/EDPutToken.h"
+#include "FWCore/Utilities/interface/ESGetToken.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DataFormats/Common/interface/Handle.h"
+
+#include "SimTracker/TrackTriggerAssociation/interface/TTTypes.h"
+
+#include <string>
+#include <map>
+#include <set>
+#include <vector>
+#include <utility>
+#include <algorithm>
+
+using namespace std;
+using namespace edm;
+
+namespace tt {
+
+  /*! \class  tt::CleanAssoc
+   *  \brief  creates TTClusterAssociationMap with clean TPs
```

Comment:
This comment describing the class is far too short. e.g. What is a clean TP? 
And does this mean we'll now have two copies of the TTClusterAssociationMap in MC samples? When is this class run?

Reply:
This is not run during Production. I run this when making a trimmed copy from grid to my local pc.

Reply:
I guess it is best to remove those files from the PR, they ended up here by accident.

Reply:
will be obsolete soon